js/pulse.js

```diff
@@ -106,7 +106,6 @@ var newProjectForm = {
     newProjectForm.formContainer.style.display = 'none';
     newProjectForm.projectContainer.style.display = 'block';
     newProjectForm.toggleFormButton.style.transform = 'rotate(0deg)';
-    setTimeout(function(){ PulseMaker.getData(); }, 250);
```

Comment:
I will clean up the refresh & notification logic after we have decided how it should work 😉 We have a few tickets open for this discussion!